.github/workflows/daily-alpha.yml

```diff
@@ -21,15 +21,15 @@ jobs:
 
       - name: Install deps
         run: |
-          npm i --location=global semver
-          npm install
-
+          yarn install
           sudo apt-get install -y ripgrep
+          cd packages/bump-version-for-cron && yarn build
         env:
           NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - name: Bump version
         id: version
-        run: npm run bump-version-for-cron
```

Comment:
Where did this come from before?

Reply:
This came from the extensions which have this as a script instead of a package.